std_msgs/Float32 view
---
string result